lib/linguist/languages.yml

```diff
@@ -5580,6 +5580,14 @@ sed:
   ace_mode: text
   tm_scope: source.sed
   language_id: 847830017
+vvvv:
```

Comment:
Change the type to `data`, please.

Reply:
I'd rather we simply add it as an XML extension. It'll achieve the same result.

Reply:
Just so I understand it, what effect would it have to add it as an XML extension?

Using gitattributes overrides is of course pointless, because I want all vvvv projects to be visible as such and not just mine.

Reply:
> Just so I understand it, what effect would it have to add it as an XML extension?

It would be correctly syntax highlighted and would be "searchable" as XML.